pyproject.toml

```diff
@@ -18,7 +18,7 @@ dependencies=[
     "google-cloud-bigquery",
     "google-cloud-bigquery-storage",
     "statsmodels",
-    "marginaleffects==0.0.11",
+    "marginaleffects>=0.0.11",
```

Comment:
If it was pinned before, do we have any concerns about updating it now? If not, can we just remove the `>=0.0.11` part?

Reply:
Daniel is out and I need to ask him about it, I vaguely remember something about having to pin this

Reply:
lol there is a whole PR just to pin it so I'll revert: https://github.com/mozilla/mozanalysis/pull/227

Reply:
fixed

Reply:
Interesting. Version 0.0.11 (which we're pinned to) was released May 5, then the PR was May 15 (previously unpinned), then version 0.0.12 was released May 31. I'm guessing the pinning was the force an update as opposed to prevent a regression.

In any case, I think it's fine to leave it alone for now and ask Daniel when he's back if he remembers the context around that PR.